--- DRAMSim2-orig/CSVWriter.h	2012-04-26 13:34:55.838903168 -0600
+++ DRAMSim2/CSVWriter.h	2012-04-26 17:02:08.550873261 -0600
@@ -92,7 +92,7 @@
 				}
 				char tmp_str[MAX_TMP_STR]; 
 				snprintf(tmp_str, MAX_TMP_STR,"%s[%u]", baseName, channel); 
-				str = strndup(tmp_str, MAX_TMP_STR); 
+				str = strdup(tmp_str); 
 			}
 			IndexedName(const char *baseName, unsigned channel, unsigned rank)
 			{
@@ -103,7 +103,7 @@
 				}
 				char tmp_str[MAX_TMP_STR]; 
 				snprintf(tmp_str, MAX_TMP_STR,"%s[%u][%u]", baseName, channel, rank); 
-				str = strndup(tmp_str, MAX_TMP_STR); 
+				str = strdup(tmp_str); 
 			}
 			IndexedName(const char *baseName, unsigned channel, unsigned rank, unsigned bank)
 			{
@@ -114,7 +114,7 @@
 				}
 				char tmp_str[MAX_TMP_STR]; 
 				snprintf(tmp_str, MAX_TMP_STR,"%s[%u][%u][%u]", baseName, channel, rank, bank); 
-				str = strndup(tmp_str, MAX_TMP_STR); 
+				str = strdup(tmp_str); 
 			}
 
 			virtual ~IndexedName()
