diff -c -r gem5-patched-v003-orig/src/base/trace.cc gem5-patched-v003/src/base/trace.cc
*** gem5-patched-v003-orig/src/base/trace.cc	2011-09-27 13:23:07.000000000 -0600
--- gem5-patched-v003/src/base/trace.cc	2012-02-02 11:27:47.517009846 -0700
***************
*** 33,38 ****
--- 33,39 ----
  #include <fstream>
  #include <iostream>
  #include <string>
+ #include <stdlib.h>
  
  #include "base/misc.hh"
  #include "base/output.hh"
