--- SConscript	2012-05-22 16:06:49.841769160 -0600
+++ SConscript	2012-05-01 10:22:50.721327263 -0600
@@ -959,8 +959,8 @@
 # Optimized binary
 makeEnv('opt', '.o',
         CCFLAGS = Split(ccflags['opt']),
-        CPPDEFINES = ['TRACING_ON=1','CONFIG_SST=1'])
+        CPPDEFINES = ['TRACING_ON=1','CONFIG_SST=1','USE_SST_DEVICE=1'])
         #CPPDEFINES = ['TRACING_ON=1'])
 
 # "Fast" binary
 makeEnv('fast', '.fo', strip = True,
