diff -r 1c918cab85bc src/dll/DerivO3CPU.cc
--- a/src/dll/DerivO3CPU.cc	Mon Dec 16 16:28:41 2013 -0700
+++ b/src/dll/DerivO3CPU.cc	Mon Feb 10 10:31:57 2014 -0700
@@ -248,6 +248,7 @@
     PMparams.megsMem = PMparams.range.size() / (1024 * 1024);
     PMparams.ddrConfig = params.find_string( "ddrConfig", "" );
     PMparams.tx_q = params.find_integer( "tx_q", 0 );
+    PMparams.null = 1;
 
 	bool useFaux = (bool)params.find_integer("use_external", 0);
 
diff -r 1c918cab85bc src/dll/fauxPhysicalMemory.cc
--- a/src/dll/fauxPhysicalMemory.cc	Mon Dec 16 16:28:41 2013 -0700
+++ b/src/dll/fauxPhysicalMemory.cc	Mon Feb 10 10:31:57 2014 -0700
@@ -52,8 +52,9 @@
 	PhysicalMemory(p)
 {
 	/* Get rid of the mapped memory - just a waste of space */
-
-	munmap((char*)pmemAddr, size());
+	if ( pmemAddr ) {
+	    munmap((char*)pmemAddr, size());
+	}    
 	pmemAddr = NULL;
 }
 
