diff -c -r gem5-patched-v004-orig/src/base/trace.cc gem5-patched-v004/src/base/trace.cc
*** gem5-patched-v004-orig/src/base/trace.cc	2011-09-27 13:23:07.000000000 -0600
--- gem5-patched-v004/src/base/trace.cc	2012-02-02 12:50:58.596168476 -0700
***************
*** 33,38 ****
--- 33,39 ----
  #include <fstream>
  #include <iostream>
  #include <string>
+ #include <stdlib.h>
  
  #include "base/misc.hh"
  #include "base/output.hh"
diff -c -r gem5-patched-v004-orig/src/sim/simulate.cc gem5-patched-v004/src/sim/simulate.cc
*** gem5-patched-v004-orig/src/sim/simulate.cc	2012-01-17 17:49:39.000000000 -0700
--- gem5-patched-v004/src/sim/simulate.cc	2012-02-02 10:10:26.237855845 -0700
***************
*** 29,34 ****
--- 29,35 ----
   *          Steve Reinhardt
   */
  
+ #include <cstdio>
  #include "base/misc.hh"
  #include "base/pollevent.hh"
  #include "base/types.hh"
