diff -Naur sst/elements/macsimComponent/macsimComponent.cpp sst/elements/macsimComponent/macsimComponent.cpp
--- sst/elements/macsimComponent/macsimComponent.cpp	2013-02-14 10:51:46.000000000 -0700
+++ sst/elements/macsimComponent/macsimComponent.cpp	2013-08-27 10:25:49.371953703 -0600
@@ -1,14 +1,16 @@
 #include "sst_config.h"
+#include "sst/core/serialization.h"
+#include "macsimComponent.h"
 
-#include "sst/core/serialization/element.h"
+#include "sst/core/debug.h"
 #include "sst/core/element.h"
+#include "sst/core/params.h"
 
-#include "macsimComponent.h"
 
 #include <sys/time.h>
 using namespace SST;
 
-macsimComponent::macsimComponent(ComponentId_t id, Params_t& params) : Component(id) {
+macsimComponent::macsimComponent(ComponentId_t id, Params& params) : Component(id) {
 	
 	//GET PARAMETERS
 
@@ -47,11 +49,12 @@
 
 macsimComponent::macsimComponent() : Component(-1) {} //for serialization only
 
-int macsimComponent::Setup() {
+void macsimComponent::setup() {
 
 	//Ensure that this component must finish before SST kernel can terminate
-	registerExit();
-
+//	registerExit();
+	registerAsPrimaryComponent();
+	primaryComponentDoNotEndSim();
 	printf("Initializing macsim simulation state\n");
 	
 	//Format for initialization protocol
@@ -66,15 +69,13 @@
 	//cleanup
 	delete argv[0]; delete argv[1]; delete argv[2];
 
-	return 0;
 }
 
-int macsimComponent::Finish() {
+void macsimComponent::finish() {
 	printf("Simulation Finished, Finalizing");
 
   macsim->finalize();
 
-	return 0;
 }
 
 /*******************************************************
@@ -107,7 +108,7 @@
 BOOST_CLASS_EXPORT(macsimComponent)
 
 static Component* create_macsimComponent(SST::ComponentId_t id,
-                                         SST::Component::Params_t& params)
+                                         SST::Params& params)
 {
 	return new macsimComponent( id, params );
 }
diff -Naur sst/elements/macsimComponent/macsimComponent.h sst/elements/macsimComponent/macsimComponent.h
--- sst/elements/macsimComponent/macsimComponent.h	2013-02-14 10:51:46.000000000 -0700
+++ sst/elements/macsimComponent/macsimComponent.h	2013-08-27 10:23:36.930882188 -0600
@@ -11,9 +11,10 @@
 class macsimComponent : public SST::Component {
 	
 	public:
-		macsimComponent(SST::ComponentId_t id, SST::Component::Params_t& params);
-		int Setup();
-		int Finish();
+		macsimComponent(SST::ComponentId_t id, SST::Params& params);
+		void setup();
+		void finish();
+
 		
 	private:
 		macsimComponent();   // for serialization only
diff -Naur sst/elements/macsimComponent/Makefile.am sst/elements/macsimComponent/Makefile.am
--- sst/elements/macsimComponent/Makefile.am	2013-02-20 12:44:28.000000000 -0700
+++ sst/elements/macsimComponent/Makefile.am	2013-08-26 10:23:57.403334922 -0600
@@ -20,8 +20,8 @@
 SUBDIRS = scripts
 
 compdir = $(pkglibdir)
-comp_LTLIBRARIES = libmacsim.la
-libmacsim_la_SOURCES =                                        \
+comp_LTLIBRARIES = libmacsimComponent.la
+libmacsimComponent_la_SOURCES =                                        \
 macsimComponent.cpp          macsimComponent.h                         \
                                                                        \
 src/all_knobs.cc             src/all_knobs.h                           \
@@ -84,5 +84,5 @@
 
 EXTRA_DIST = 
 
-libmacsim_la_LDFLAGS = -module -avoid-version
-libmacsim_la_LIBADD = -lz
+libmacsimComponent_la_LDFLAGS = -module -avoid-version
+libmacsimComponent_la_LIBADD = -lz
diff -Naur sst/elements/macsimComponent/sst-unit-test/unit-test.py sst/elements/macsimComponent/sst-unit-test/unit-test.py
--- sst/elements/macsimComponent/sst-unit-test/unit-test.py	2013-02-14 11:12:45.000000000 -0700
+++ sst/elements/macsimComponent/sst-unit-test/unit-test.py	2013-08-26 10:23:57.403334922 -0600
@@ -94,8 +94,8 @@
     """
     cleanup
     """
-    os.system('rm -f trace_file_list NULL trace_debug.out')
-    os.system('rm -rf results')
+##    os.system('rm -f trace_file_list NULL trace_debug.out')
+##    os.system('rm -rf results')
 
 
 if __name__ == '__main__':
