diff --git src/base/param.h src/base/param.h
index c626ee4..6df87a2 100644
--- src/base/param.h
+++ src/base/param.h
@@ -88,7 +88,7 @@ class _ParamChainer {

       if (_config[_name]) {
         try {
-          return _config[_name].as<T>();
+          return _config[_name].template as<T>();
         } catch (const YAML::BadConversion& e) {
           throw ConfigurationError("Failed to parse Param \"{}\" for implementation \"{}\".", _name_prefix + _name, _impl_name);
         }
